### DISCUSSION TITLE
Requirement to add comments when reviewing and selecting "Request Changes" should be removed

### DISCUSSION THREAD

**Discussion by shelmire** [community/community#56783] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Sometime fairly recently, an error was added when submitting a code review with "Request Changes" if you leave the overall comments empty ("You need to leave a comment indicating the requested changes."). The problems is that I often leave comments on individual lines or files (not the "pending comments" as part of a review, but instantly published). Why? Because sometimes I get interrupted while doing reviews and then the feedback is unnecessarily delayed. Or maybe I'll forget to hit publish. These comments do not seem to count as part of a review, and if those are all that are present, you can't submit a review. So typically the solution is to write "see comments" which is just unnecessary spam and annoying.

Additionally this breaks another common workflow where code reviews are done live, or comments provided via chat or discussion outside of Github. In those cases, I should still be able to Request Changes. After all, we get to decide our workflow - it should not be dictated by Github in this way.

**Comment by andreahlert** [comment]
Hey there 🙂
I get the friction here. If inline comments already explain the issue, forcing an extra summary can feel redundant.

At the same time, do you think GitHub added this to ensure accountability and clarity for audit trails? Maybe the ideal solution is not removing it entirely, but making it configurable at the repository or organization level.
